### PR TITLE
Codecov added

### LINKED ISSUE #640: Addition of codecov for analyisng test coverage
We can get a reporting dashboard of the tests being run in the repo during Travis, and then can have additional quality checks on the basis of coverage
@vringar

### PR DESCRIPTION
**Pull request by shubhank-saxena**
This fixes #640

### CODE REVIEW

File: environment.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ channels:
 dependencies:
 - autopep8=1.5.2
 - beautifulsoup4=4.9.0
+- codecov=2.1.0
```
Comment (birdsarah) [inline environment.yaml:7]: See instructions: https://github.com/mozilla/OpenWPM/#managing-requirements
This file should not be manually edited.

**Comment by birdsarah** [comment]
Issue opened on #662.

File: environment.yaml [hunk 2]
```diff
@@ -15,10 +16,10 @@ dependencies:
 - nodejs=13.13.0
 - pandas=1.0.3
 - pillow=7.1.2
-- pip=20.1
+- pip=20.1.1
```
Comment (birdsarah) [inline environment.yaml:19]: @englehardt this is the kind of thing that will happen when a user adds a dependency and then needs to run repin to get it included in a PR. This same thing would have happened with requirements.in / piptools, so I think it's ok.

Comment (englehardt) [reply environment.yaml:19]: yes the same thing happened, and I agree that it's okay.

**Comment by shubhank-saxena** [comment]
@birdsarah it's done! Kindly let me know if any further changes are required!

**Comment by birdsarah** [comment]
Where can I see the codecov report that's being upload (or maybe failed to upload) that this PR generates.

**Comment by shubhank-saxena** [comment]
@birdsarah need to activate Codecov on this repo. There was a way to check on personal account few months back. But I think it's been revoked by codecov. Let me give it one more try to see if I could generate report for my fork

**Comment by birdsarah** [comment]
Thanks @shubhank-saxena. I think @englehardt needs to finalize merging this PR as he's the one that can turn on the integration.

**Comment by englehardt** [comment]
@shubhank-saxena @birdsarah Looks like this is now enabled. Can you help me understand what's necessary to see a report?

**Comment by shubhank-saxena** [comment]
> @shubhank-saxena @birdsarah Looks like this is now enabled. Can you help me understand what's necessary to see a report?

@englehardt as you can see above your comment that there was an automated report generated by codecov. Also, as soon as you commit some changes, it will reflect here in the form of similar comments.

**Pull request merged**